src/init.js

```diff
@@ -80,50 +80,17 @@ EventBus.on('signaling-recording-status-changed', ([token, status]) => {
 /**
  * Migrate localStorage to @nextcloud/browser-storage
  *
- * In order to preserve the user settings while migrating to the abstraction,
- * we loop over the localStorage entries and add the matching ones to the
- * BrowserStorage
+ * We assume migration is done from now on, so we remove the migration flag
+ * REMOVE: in stable +33
```

Comment:
+33 feels like an unar plus in a positive number.

Like +33°C ☀️ 

```suggestion
 * REMOVE: in stable 33+
```